lib/lotus/model/migrator/postgres_adapter.rb

```diff
@@ -25,7 +25,7 @@ class PostgresAdapter < Adapter
         # @since 0.4.0
         # @api private
         def create
-          new_connection.run %(CREATE DATABASE "#{ database }"#{ create_options })
```

Comment:
why this doesn't run with postgresql? `createdb` is a wrapper for `CREATE DATABASE`